client/modules/IDE/components/EditorAccessibility.js

```diff
@@ -9,7 +9,7 @@ class EditorAccessibility extends React.Component {
     if (this.props.lintMessages.length > 0) {
       for (let i = 0; i < this.props.lintMessages.length; i++) {
```

Comment:
I didn't notice this earlier, but these days in JS Land I think it's better practice to use forEach than a for loop.